data/sources/unassigned.json

```diff
@@ -1,6 +1,5 @@
 {
   "164083100": "Source: Display of Supremacy, Weekly Challenge",
-  "2939318890": "Source: Wayfinder's Compass",
   "3094114967": "Source: Season of the Lost ritual playlists",
   "3100439379": "Source: Mission Exorcism",
   "3845969330": "Source: Xûr"
```

Comment:
should we add this?

Reply:
that's weird really weird. i feel strange about what items are marked as 3845969330